src/index.html

```diff
@@ -11,6 +11,25 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="search-form search-form__big" data-qa="big">
+      <label class="search-label search-label__big" for="search">
+        <input
```

Comment:
`for` attribute is redundant here as input is nested inside the label. It should be used with id of input if needed
https://css.in.ua/html/tag/label/for